scripts/create-container-registry-creds.sh

```diff
@@ -13,7 +14,9 @@ if [[ "${CI:-}" == "true" ]]; then
     CTR_REGISTRY_PASSWORD=${DOCKER_PASS:-}
 fi
 
-[[ -z "$CTR_REGISTRY_PASSWORD" ]] && exit 0
+if [ "$USE_PRIVATE_REGISTRY" = false ]; then
```

Comment:
Are we standardizing on POSIX compliant tests (`[ ]`), or the bash extension (`[[ ]]`)? It looks like it is mixed. The latter is more common, and although less compatible I think it's fine as long as it is consistent.

Reply:
I also think we should quote `"false"` so that it is more obvious we are dealing with strings, and viewers don't confuse it with the `false` bin (`/usr/bin/false`).

Reply:
@trstringer Good point, I've honestly seen both used across our scripts, but for the sake of consistency I can use `[[ ]]` here.